app/src/main/java/co/nimblehq/compose/ecommerce/ui/screen/home/Home.kt

```diff
@@ -0,0 +1,41 @@
+package co.nimblehq.compose.ecommerce.ui.screen.home
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+import co.nimblehq.compose.ecommerce.ui.theme.Purple600
+
+@Composable
+fun HomeScreen() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = Color.White)
+            .wrapContentSize(Alignment.Center)
+    ) {
+        Text(
+            text = "Home View",
+            fontWeight = FontWeight.Bold,
+            color = Purple600,
+            modifier = Modifier.align(Alignment.CenterHorizontally),
```

Comment:
Do we need to set `Alignment.CenterHorizontally`?

Reply:
The purpose is to display the dummy text at the center of the screen. We will update this in the next PR accordingly 😄